app/views/pages/get-started/working-with-the-code-and-files.liquid

```diff
@@ -0,0 +1,178 @@
+
+If you’re following this guide, you should be able to find a file named index.html.liquid in your codebase. Open it up in your code editor of choice and change any line. For example let’s locate a line with the welcome message:
+
+```html
+Welcome to your Demo instance!
+```
+
+Try changing it into something more personal like:
+
+```html
+This is my first platfromOS instance!
+```
+
+Save the file and – as we learned – send the codebase to the instance by running the deploy command in the command line again:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy dev
+</code></pre>
+
+After visiting your instance URL or refreshing previously opened page, you should see the new message there.
+
+Ok, this works, but having to manually deploy each change would take far too much time. You wouldn’t want to enter any commands or wait so long for the pos-cli to send all of the files in your codebase each time you change a single line of your code. Because this is what the deploy command is doing – sending all the files in the codebase to the instance.
+
+
+## Upload code changes automatically when you save the file
+
+On a daily basis, you want the instance to immediately reflect the changes you made in the code with a minimum effort on your side.
+
+This process we call **syncing** and it’s handled by a pos-cli command ran from the root project’s directory:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync &lt;environment&gt;
+</code></pre>
+
+When executing this command you will learn that it does not end by itself. This is a command that you want to run and leave opened while you make code changes.
+
+What `pos-cli sync` is doing is basically observing your codebase, so when you save changes in a file – it automatically uploads it to your instance. Just the file that changed so it’s quick and efficient!
+
+
+### Syncing in action
+
+For our example we can try another change in the code, but before we’ll do it – let’s start the sync using the following command:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli sync dev
+</code></pre>
+
+A confirmation message saying that the synchronization has been started should appear, so we can find another place in the code to test how this works. Please find the `Hello world` title in `index.html.liquid` of the demo page and change it to something else, like `Hello beautiful world`.
+
+After the change, just save the file.
+
+In the command line you should see a confirmation message saying that the edited page was synced. Refreshing the page in the browser should confirm that everything worked and the new text content is in place.
+
+Syncing handles not only code changes, but also all the other assets. Images, JavaScript files or even GraphQL queries – any change in your project directories will be automatically reflected on your instance.
```

Comment:
Might be worth saying that it will also delete a file, handle rename etc. If there's any syntax error the pos-cli will display error message with the error and the file will not be synced?